ui/app/components/tool-actions-form.js

```diff
@@ -135,14 +135,11 @@ export default Component.extend(DEFAULTS, {
       this.reset();
     },
 
-    updateTtl(evt) {
-      const ttl = evt.enabled ? `${evt.seconds}s` : '30m';
+    updateTtl(ttl) {
       set(this, 'wrapTTL', ttl);
     },
 
-    codemirrorUpdated(val, codemirror) {
-      codemirror.performLint();
```

Comment:
codemirror things need to happen inside the component where codemirroris being used, thus the property `codemirror` is used in the tool-hash component.